Doc/library/sqlite3.rst

```diff
@@ -250,11 +258,12 @@ Module functions and constants
    *detect_types* parameter and the using custom **converters** registered with the
    module-level :func:`register_converter` function allow you to easily do that.
 
-   *detect_types* defaults to 0 (i. e. off, no type detection), you can set it to
-   any combination of :const:`PARSE_DECLTYPES` and :const:`PARSE_COLNAMES` to turn
-   type detection on. Due to SQLite behaviour, types can't be detected for generated
-   fields (for example ``max(data)``), even when *detect_types* parameter is set. In
-   such case, the returned type is :class:`str`.
+   *detect_types* defaults to 0 (type detection disabled).
+   Set it to any combination of :const:`PARSE_DECLTYPES` and
```

Comment:
If both get passed and the column has both a data type in the query and a declared data type in the underlying SQLite table, that both match to conversion functions in the lookup table, which takes priority? I'm guessing the former?

Reply:
I'll clarify. Thanks!